tests/integration/test_design.py

```diff
@@ -382,7 +382,10 @@ def test_add_component_with_instance_name(modeler: Modeler):
     nested_component = design.add_component("NestedComponent")
     nested_component2 = design.add_component("NestedComponent2", instance_name="first instance")
 
+    assert nested_component.name == "NestedComponent"
```

Comment:
kind of unnecessary since we already have other tests cover this. but whatever @RobPasMue

Reply:
When discussing with @b-matteo, we saw that name and instance_name were being overwritten in the server side by whatever was passed as instance_name. It would be interesting to see that the message returned from the service is consistent with what we expect and that no overriding of the name takes place. That is why we are actively checking as well both name and instance_name.

Tests are not expected to be "performant" but make the code reliable. The more things we can check (that have been demonstrated to cause an impact), the better.